src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -27,6 +27,9 @@
 #include <functional>
 #include <boost/format.hpp>
 
+/* exposed by shim */
+int xclUpdateSchedulerStat(xclDeviceHandle);
```

Comment:
Please include shim.h

Reply:
"xclUpdateSchedulerStat" this function is not declared in shim.h file.  Hence including shim.h is not required I believe, Please let me know if my understanding is correct.

Reply:
Sure, ok.  But instead of declaring it here, it should be declared in shim.h since it is defined in shim.cpp, but I'm okay leaving it the way you have it.

Reply:
Fixed this issue